database/databasePostgresql/schema.sql

```diff
@@ -17,8 +16,7 @@ CREATE TABLE places (
 );
 
 CREATE TABLE users (
-  user_id_serial serial PRIMARY KEY,
-  user_id uuid,
+  id SERIAL NOT NULL PRIMARY KEY,
   first_name varchar(25),
   last_name varchar(25),
   address_line_1 varchar(50),
```

Comment:
Hopefully it works good for you, but you can use TEXT instead of varchar(50) for addresses , so you won't think about fakers address length, which can possibly be longer, but I'm not sure.

Reply:
Yeah, this worked for me. The text might be easier since we don't need to worry about the length, and performance are same on both varchar and text.